eng/scripts/TypeSpec-Requirement.ps1

```diff
@@ -12,70 +12,19 @@ Set-StrictMode -Version 3
 . $PSScriptRoot/ChangedFiles-Functions.ps1
 . $PSScriptRoot/Logging-Functions.ps1
 
-$script:psYamlInstalled = $false
```

Comment:
Logic moved to `/eng/tools/suppressions/src`.

Reply:
For my context what were the main reasons you moved this logic from PS to TS? I'm not necessarily opposed to it but just wanted to understand what things made you decide to move it.

Reply:
We expect the suppressions code will be called from both PS and TS consumers, so from this perspective either langauge is equally valid.

The features we wanted were growing in complexity, and several TS modules were helpful:

* `minimatch`: Processes "globstar" wildcards (used by other tools like git)
* `zod`, `zod-validation-error`: Validate parsed yaml matches expected schema, generates user-friendly errors

PS might have similar modules, but at least in `azure-rest-api-specs`, managing `node` modules is generally easier than `pwsh` modules, since we already have a global `package-lock.json`.  PS modules need to be installed at runtime.

Reply:
Thanks for the context.